dist/image-sequencer.js

```diff
@@ -47721,6 +47721,25 @@ ImageSequencer = function ImageSequencer(options) {
     else modulesdata = modules[name][1];
     return modulesdata;
   }
+
+  function toString(step_name,options){
+    if(step_name){
```

Comment:
I'd like if we can start maintaining a tighter formatting -- like `function functionName(a, b) {` and then there won't be as much change when we do auto-formatting. Maybe worth doing a run of auto-formatting in a PR before moving forward?